src/test/data/trivially_invalid.json

```diff
@@ -1,57 +1,52 @@
     "Transaction with operator reward set to uint16_t numerical limit"
 ],
-[
-    "9827a1ccbf225f9e4faf4db78d2529ee611d133e9cd2974446b0a38a409aa957",
-    "proupregtx",
-    "03000100010000000000000000000000000000000000000000000000000000000000000000ffffffff016affffffff0100f2052a01000000016a00000000fd12010100000000003c55913dfbc2dc0fcbf97f992550685a39980a6f0de715352ee05af52b466b330100000000000000000000000000ffff05bd9f5e270f03e6d2bd33e88835e8995fc6d570f30bd24e20dd0074a43322d7c92ca3523b14f4eedc97f1b083977ac21d7483d9b34f32782e76b3e3bfe2bd089794f927d6443796d05d03e6d2bd33e88835e8995fc6d570f30bd24e20dd00001976a914f95848adc4ae65097342682646d98a1469e8597488ac41cf0fbc60088337c47157528fff425645879561f6628c9c095bde959bef3abc416666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666",
-    "Valid proregtx transaction mislabeled as proupregtx"
-],
 [
     "299767699266aa16f38176037c393e093f03dbfa5429e76ad4f756f70b22ff5b",
     "proupregtx",
     "03000300010000000000000000000000000000000000000000000000000000000000000000ffffffff016affffffff0100f2052a01000000016a00000000e40100b2d3e8dea42b9c8331472962bb36dea7b684b0f6dc8f5869c80fc24936199a040000934e4e541c310c2d17df7b48040d6ae9446fab9530abcbebdd1b46d847a5b4f8898057060143436dc7003e1e6fd9bf4500000000000000000000000000000000000000001976a914b9a701d799b3d96b91a4d0311fb39c29ebad98ed88ac280b34567afd60136868e50b86db2c7e35510ed9072fe34439b9e665f9cc6492416666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666",
+    "bad-protx-key-null",
     "Transaction with invalid keyIDVoting"
 ],
 [
     "95cb5e1af901021c776b9dc43147924502e05301ba9aa9e32e9eb0abc1daa910",
     "proupregtx",
     "03000300010000000000000000000000000000000000000000000000000000000000000000ffffffff016affffffff0100f2052a01000000016a00000000e40000bfd280d22a21574ea9f568f7a729b370f4051b89eab6df3d0d74b65fde1bdb7900000a6a7fb814db71eb6ef12341749d81b5fd326eb4b9445a3be65668da9856910b378ec6427fe18fc81c39499e7c7060647618f8ec94404051e4a7ab68f1b017fd9e7f94761976a914c18e0b18e0d992e7471d8a4d4da428e8e191328788ac959ca83b61eeb3f5543973253da6da8e9ac23f9e7844453ac5543b51d0177799416666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666",
+    "GetTxPayload",
     "Transaction with version set to null"
 ],
-[
-    "943d178782e22b573897d8d1835ac388d80d2ca5d6efcc72b32d7d83940a3c5d",
-    "proupregtx",
-    "03000400010000000000000000000000000000000000000000000000000000000000000000ffffffff016affffffff0100f2052a01000000016a00000000a40100886e39ad019694721b7ab5b5523b5f6374cc7c322410681c7f897ab9bc4098be00000c5d4e38678fee4f0e2386609f31f40a41390a9fdc65bfa8e54157c84123eede9308be92875d1c8f52f9b28215dbc95b3006f0cc8e2f410ced1ed1d6722287978ecbab0a19ab84d6025dfb6dfaa27d2a0dc0180833577674621930a3ea6fd44e3f461dd9b5b6f79c786958d3e6045c2d9eb57629c2c2daf5403690b61ccdfe94",
```

Comment:
Why are these test vectors removed?

Reply:
bcs it tests nothing useful. It initialized by binary presentation since now instead `proupregtx` string - so, it doesn't actually test anything important. So, if transaction successfully deserialized -> it means that type determined correctly.

Reply:
Seems good to me